src/src/components/dialog/Dialog.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div
+    v-if="dialog.show"
+    class="dialog-container"
+    >
+    <div class="dialog-bubble">
+        <fa
+            icon="sort-up"
+            class="dialog-top-icon" />
+        <div class="dialog-bubble-content">
+            <div class="dialog-header">
+                <h2>My Profile</h2>
+            </div>
+            <div class="dialog-main">
+            <div class="profile-avatar-container">
+                <div class="avatar">
+                <img :src="profileData.avatar" class="avatar-image" :alt="profileData.name">
+                </div>
+                <div class="avatar-info">
+                    {{ profileData.fullName }}
+                    <sub>{{ profileData.email }}</sub>
+                </div>
+            </div>
+            </div>
+            <div class="dialog-footer">
+                <a href="" v-if="dialog.settings">Profile Settings</a>
```

Comment:
Codacy has a fix for the issue: [Attribute "v-if" should go before "href".](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
                <a v-if="dialog.settings" href="">Profile Settings</a>
```